src/vea/vea_free.c

```diff
@@ -447,7 +447,6 @@ migrate_end_cb(void *data, bool noop)
 		if (vsi->vsi_unmap_ctxt.vnc_unmap != NULL) {
 			D_ALLOC_PTR(vue);
 			if (vue == NULL) {
-				rc = -DER_NOMEM;
```

Comment:
Added @NiuYawei to review this one.

Reply:
Yeah, this "rc = -DER_NOMEM" looks not necessary to me, it can be removed.